quimb/gen/operators.py

```diff
@@ -343,17 +343,63 @@ def fsim(theta, phi, dtype=complex, **kwargs):
     a = cos(theta)
     b = -1j * sin(theta)
     c = exp(-1j * phi)
-
     gate = [[1, 0, 0, 0],
             [0, a, b, 0],
             [0, b, a, 0],
             [0, 0, 0, c]]
 
+
+
     gate = qu(gate, dtype=dtype, **kwargs)
     make_immutable(gate)
     return gate
 
 
+#https://arxiv.org/pdf/2010.07965.pdf, Eq. 18
```

Comment:
This kind of info can/should go in the docstring! Then you don't need o browse the source code to see it: e.g.
```python
qu.fsimg?
```
in ipython shows the docstring.